### PR TITLE
Converted FadingTextView file from Java to Kotlin

### PR DESCRIPTION
**Pull request by ebayraktar**
This commit includes FadingTextView Kotlin file and build gradle upgrades

### CODE REVIEW

File: fadingtextview/build.gradle [hunk 1]
```diff
@@ -47,6 +48,5 @@ dependencies {
     testImplementation 'junit:junit:4.12'
 }
 
-apply from: 'https://raw.githubusercontent.com/nuuneoi/JCenter/master/installv1.gradle'
-apply from: 'https://raw.githubusercontent.com/nuuneoi/JCenter/master/bintrayv1.gradle'
-
+//apply from: 'https://raw.githubusercontent.com/nuuneoi/JCenter/master/installv1.gradle' // deprecated
```
Comment (rosenpin) [inline fadingtextview/build.gradle]: you can just remove it if it's deprecated. no need to keep it commented out

File: fadingtextview/src/main/java/com/tomer/fadingtextview/FadingTextView.kt [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+    context: Context,
+    attrs: AttributeSet? = null,
+    defStyleAttr: Int = 0
+) : AppCompatTextView(context, attrs, defStyleAttr) {
+
+    private val fadeInAnimation: Animation by lazy { AnimationUtils.loadAnimation(context, R.anim.fadein) }
+    private val fadeOutAnimation: Animation by lazy { AnimationUtils.loadAnimation(context, R.anim.fadeout) }
+    private val handler: Handler = Handler(Looper.getMainLooper())
+
+    var texts: Array<CharSequence> = arrayOf()
```
Comment (rosenpin) [inline fadingtextview/src/main/java/com/tomer/fadingtextview/FadingTextView.kt]: use `emptyArray` instead

File: fadingtextview/src/main/java/com/tomer/fadingtextview/FadingTextView.kt [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+    }
+
+    override fun startAnimation(animation: Animation) {
+        if (isShown && !stopped) {
+            super.startAnimation(animation)
+        }
+    }
+
+    protected fun startAnimation() {
+        if (!isInEditMode) {
```
Comment (rosenpin) [inline fadingtextview/src/main/java/com/tomer/fadingtextview/FadingTextView.kt]: probably unrelated, but replace with 
``` if (isInEditMode) return```

File: app/build.gradle [hunk 4]
```diff
@@ -1,8 +1,8 @@
 apply plugin: 'com.android.application'
```
Comment (rosenpin) [inline app/build.gradle]: this could be a good chance to move from gradle to kts if you're up for it. not necessary for this PR though

**Review by rosenpin — COMMENTED** [review]
Thank you for the PR!  
I left a few comments

**Comment by ebayraktar** [comment]
Thanks for the review, I had some changes and build gradles migrate to Kotlin DSL

**Comment by rosenpin** [comment]
Thanks, looking good  
I'll test and merge it

**Comment by rosenpin** [comment]
Looking good! Thanks again!

**Pull request merged**